resource/ckb-miner.toml

```diff
@@ -32,11 +30,22 @@ dsn = "" # {{
 # testnet => dsn = "https://48c6a88d92e246478e2d53b5917a887c@sentry.io/1422795"
 # }}
 
-[miner]
+[miner.client]
 rpc_url = "http://127.0.0.1:8114/" # {{
 # _ => rpc_url = "http://127.0.0.1:{rpc_port}/"
 # }}
 block_on_submit = true
 
 # block template polling interval in milliseconds
 poll_interval = 1000
+
+[[miner.workers]]
+worker_type = "CuckooSimple"
+threads     = "1"
```

Comment:
Why not integer?

Reply:
We need to support `bool` or `float` here, for example: 
```
[[miner.workers]]
worker_type = "CuckooCpuMean"
trim        = "true"

[[miner.workers]]
worker_type = "Dummy"
delay_type  = "Normal"
mean        =  "5000"
std_dev     =  "32.5"
```

Reply:
You can use `toml::Value` as the hashmap value.

Reply:
Another choise: use internal tagged enum

```rust
#[derive(Serialize, Deserialize)]
#[serde(tag = "worker_type")]
enum WorkerConfig {
    CuckooCpuMean { trim: bool },
    Dummy { delay_type: String, mean: u32, std_dev: f64 },
}
```

Reply:
@doitian @TheWaWaR I pushed a new commit https://github.com/nervosnetwork/ckb/pull/961/commits/4059a26365b3cda571ebc7d5cfdae9ca62662d0c to resolve this issue, please review again, thanks